Formula/kube-score.rb

```diff
@@ -0,0 +1,35 @@
+class KubeScore < Formula
+  desc "Kubernetes object analysis recommendations for improved reliability and security"
+  homepage "https://kube-score.com"
+  url "https://github.com/zegl/kube-score/archive/refs/tags/v1.13.0.tar.gz"
+  sha256 "a09198f9a49c0008f9a34b418d85d3eb1299d0f04077e423eff30f1cad8abd43"
+  license "MIT"
+  head "https://github.com/zegl/kube-score.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/kube-score"
```

Comment:
let's set some build flags for the `kube-cost version` command

 ```suggestion
    ldflags = %W[
      -s -w
      -X main.version=#{version}
      -X main.commit=#{Utils.git_head}
      -X main.date=#{time.iso8601}
    ].join(" ")
    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/kube-score"
```